### ISSUE TITLE
Enhanced <select> not working as expected in JAWS

### ISSUE THREAD

**Issue by charge-valtech** [alphagov/accessible-autocomplete#289]
This is feedback from a JAWS user of our prototype:

"This screen did not work as expected. I navigated to what JAWS announced as a combo box. No such box exists within the JAWS combo dialog box, however when I activated the edit field dialog box , it was an edit field. I typed into this edit field and nothing happened. I asked a sighted colleague to use a mouse and then visually a drop down box appeared on screen. I tried to navigate this by using my arrow keys, visually the focus moved according to my colleague, but JAWS did not announce the content. At this point I decided to leave testing the rest of the application."

This is the HTML:
```
<select name="council-name" class="enhanced-select" name="" id="councilPicker">
      {% include "includes/councils-options.html" %}
</select>
```

We've included the minified autocomplete JS and it is initiated in the page like this:
```
$(document).ready(function () {
      accessibleAutocomplete.enhanceSelectElement({
        defaultValue: '',
        showAllValues: true,
        selectElement: document.querySelector('#councilPicker')
      });
    });
```

Browser IE 11
JAWS V17

**Comment by nickcolley** [comment]
Will be in the 2.0.0 release :)
